server/udp.rb

```diff
@@ -1,9 +1,13 @@
 require 'socket'
+
 BasicSocket.do_not_reverse_lookup = true
+
 # Create socket and bind to address
 client = UDPSocket.new
-client.bind(nil, 8125)
+client.bind('0.0.0.0', 8125)
```

Comment:
Testing with docker showed that the `nil` here would bind only to localhost/loopback. From the container it needed to be `0`s.